pkg/v1/sdk/features/constants/constants.go

```diff
@@ -0,0 +1,11 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package constants
+
+const (
+	// TKGSystemFeatureGate is the name of the default system feature gate
+	TKGSystemFeatureGate = "tkg-system"
```

Comment:
Do we still need a new `constants` package? At least one of these constants is already defined here: https://github.com/vmware-tanzu/tanzu-framework/blob/986787530cee246bb2d557ef3fa6e02dd5f8a7da/pkg/v1/sdk/features/featuregate/featuregate.go#L17-L20

Reply:
We should probably just add `TKGSystemFeatureGateNamespace` to that file.

Reply:
thanks for catching this, we don't need this constants package as we are not using it anymore